src/participant/participant.ts

```diff
@@ -748,7 +761,7 @@ export default class ParticipantController {
       for (const msg of messages) {
         stream.markdown(msg);
       }
-      return { metadata: {} };
+      return { metadata: { responseType: 'empty' } };
```

Comment:
I think instead of 'empty', we could just send undefined to telemetry.

Reply:
I would prefer not to send `undefined` for expected values though I can see `empty` is not the correct value here either. I'll need to send the correct type based on the response we create in `handleEmptyQueryRequest`. I'm thinking of sending `connections/namespace` if we prompt the user for input and `no_prompt` in the default case.